### PR TITLE
Fix | Static Analyzers

### PR DESCRIPTION
**Pull request by AlexNDRmac**
Hello!

* Type: bug fix | code quality
* Link to issue:

In raising this pull request, I confirm the following:

- [X] I have checked that another pull request for this purpose does not exist
- [ ] I wrote some tests for this PR
- [ ] I updated the CHANGELOG

Small description of change:

- Improved github actions to run all checks even if some check failed

Thanks

### CODE REVIEW

File: .github/workflows/analysis.yml [hunk 1]
```diff
@@ -41,10 +41,13 @@ jobs:
           ./phpcs --version
 
       - name: Run PHP_CodeSniffer
-        run: ./phpcs
+        run: |
+          ./phpcs --runtime-set ignore_warnings_on_exit true
 
       - name: Run PHP CS Fixer
+        if: always()
```
Comment (ruudboon) [inline .github/workflows/analysis.yml:34]: Why not use:
```
continue-on-error: true
```
https://help.github.com/en/actions/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions#jobsjob_idstepscontinue-on-error

Comment (AlexNDRmac) [reply .github/workflows/analysis.yml:34]: Yes... it looks better. I'll fix this conditional, thanks :)

Comment (AlexNDRmac) [reply .github/workflows/analysis.yml:34]: But... job status always successful. Conditionals in steps doesn't change step status.
Main idea - is run all checks and see all reports from PHP tools... then developer can fix those problem with one fix commit.
So... I'll revert to explicit conditions :)

Comment (ruudboon) [reply .github/workflows/analysis.yml:34]: Ah, wasn’t expecting that status will change

File: .github/workflows/analysis.yml [hunk 2]
```diff
@@ -41,10 +41,14 @@ jobs:
...
       - name: Run PHP_CodeSniffer
-        run: ./phpcs
+        run: |
+          ./phpcs --runtime-set ignore_warnings_on_exit true
 
       - name: Run PHP CS Fixer
-        run: ./php-cs-fixer fix --diff --dry-run -v
+        if: always()
+        run: |
+          ./php-cs-fixer fix --diff --dry-run -v
```
Comment (sergeyklay) [inline .github/workflows/analysis.yml]:
```diff
-        run: |
+        run: ./php-cs-fixer fix --diff --dry-run -v
-          ./php-cs-fixer fix --diff --dry-run -v
```

File: .github/workflows/analysis.yml [hunk 3]
```diff
@@ -41,10 +41,14 @@ jobs:
           ./phpcs --version
 
       - name: Run PHP_CodeSniffer
-        run: ./phpcs
+        run: |
+          ./phpcs --runtime-set ignore_warnings_on_exit true
```
Comment (sergeyklay) [inline .github/workflows/analysis.yml]:
```diff
-        run: |
+        run: ./phpcs --runtime-set ignore_warnings_on_exit true
-          ./phpcs --runtime-set ignore_warnings_on_exit true
```

**Review by sergeyklay — CHANGES_REQUESTED** [review]

**Comment by sergeyklay** [comment]
Thank you

**Pull request merged**